src/process.rs

```diff
@@ -446,6 +447,99 @@ pub struct Io {
     pub cancelled_write_bytes: u64,
 }
 
+/// Mount information from `/proc/<pid>/mountstats`.
+#[derive(Debug, PartialEq)]
+pub struct MountStat {
+    device: Option<String>,
+    mount_point: PathBuf,
+    fs: String,
+    statistics: Option<MountNFSStatistics>,
+}
+
+impl MountStat {
+    pub fn from_reader<R: io::Read>(r: R) -> Vec<MountStat> {
+        vec![]
+    }
+}
+
+/// Only NFS mounts provide additional statistics in `MountStat` entries.
+#[derive(Debug, PartialEq)]
+pub struct MountNFSStatistics {
+    version: String,
+    // * opts: HashMap<String, Some(String)>
+    /// Duration the NFS mount has been in existence.
+    age: Duration,
+    // * fsc (?)
+    // * impl_id (NFSv4): Option<HashMap<String, Some(String)>>
+    // * caps: HashMap<String, String>
+    // * nfsv4 (NFSv4): Option<HashMap<String, Some(String)>>
+    // * sec: HashMap<String, Some(String)>
```

Comment:
Probably `opts`, `impl_id`, `caps`, `nfsv4`, and `sec` should all be the same type

Reply:
Yeah I was thinking about something like `type Options = HashMap<String, Option<String>>` but I am not really fond of the name yet.